src/tests/VisualizationInputs.test.ts

```diff
@@ -0,0 +1,504 @@
+// (C) 2019 GoodData Corporation
```

Comment:
These tests were all cut and pasted from VisualizationObject.test.ts. Only the imports were updated and some different names unified.